src/app/components/topProd.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Table, TableHeader, TableColumn, TableBody, TableRow, TableCell, Spinner, getKeyValue } from "@nextui-org/react";
+import { useInfiniteScroll } from "@nextui-org/use-infinite-scroll";
+import { useAsyncList } from "@react-stately/data";
+
+export default function TopProd() {
+  type StringNumberKey<T> = Extract<keyof T, string | number>;
```

Comment:
Stop Copying codes !

Reply:
-it's impossible 
-why???
-just impossible